src/simplejavacalculator/UI.java

```diff
@@ -95,15 +95,15 @@ public UI() throws IOException {
       butMinus = new JButton("-");      
       butMultiply = new JButton("*");      
       butDivide = new JButton("/");      
-      butEqual = new JButton("=");      
-      butSquareRoot = new JButton("sqrt");      
-      butSquare = new JButton("x*x");      
-      butOneDividedBy = new JButton("1/x");      
+      butEqual = new JButton("=");
+      butSquareRoot = new JButton("<html>&#8730;x</html>");
```

Comment:
Could we avoid having `html` tags inside JButton…? 
I wonder if there is any alternatives?